### PR TITLE
typo: convert tab to space

### PR DESCRIPTION
**Pull request by jsvisa**
Signed-off-by: Delweng <delweng@gmail.com>

**What type of PR is this?**

> Uncomment one line below and remove others.
>
> Bug fix
> Feature
> Documentation
> Other

**What does this PR do? Why is it needed?**

**Which issues(s) does this PR fix?**

Fixes #

**Other notes for review**

### CODE REVIEW

File: prysm.sh [hunk 1]
```diff
@@ -130,11 +130,11 @@ function verify() {
     fi
     checkSum="shasum -a 256"
     hash shasum 2>/dev/null || {
-	checkSum="sha256sum"
-    	hash sha256sum 2>/dev/null || {
-		echo >&2 "SHA checksum utility not available. Either install one (shasum or sha256sum) or run with PRYSM_ALLOW_UNVERIFIED_BINARIES=1."
-		exit 1
-    	}
+    checkSum="sha256sum"
```
Comment (prestonvanloon) [inline prysm.sh:133]: This indentation is off

File: prysm.sh [hunk 2]
```diff
@@ -146,7 +146,7 @@ function verify() {
     gpg --list-keys "$PRYLABS_SIGNING_KEY" >/dev/null 2>&1 || curl --silent https://prysmaticlabs.com/releases/pgp_keys.asc | gpg --import
     (
         cd "$wrapper_dir"
-	$checkSum -c "${file}.sha256" || failed_verification
+    $checkSum -c "${file}.sha256" || failed_verification
```
Comment (prestonvanloon) [inline prysm.sh:149]: This indentation is off

**Review by prestonvanloon — CHANGES_REQUESTED** [review]
Thanks for submitting this! There are a few visual differences in indentation that I think should be addressed. 

Tab vs spaces doesn't matter here, but indentation needs to read well.

**Comment by jsvisa** [comment]
@prestonvanloon  Thanks for your advice, fix them all. pls take another look.

File: prysm.sh [hunk 3]
```diff
@@ -130,11 +129,11 @@ function verify() {
...
-	checkSum="sha256sum"
-    	hash sha256sum 2>/dev/null || {
-		echo >&2 "SHA checksum utility not available. Either install one (shasum or sha256sum) or run with PRYSM_ALLOW_UNVERIFIED_BINARIES=1."
-		exit 1
-    	}
+        checkSum="sha256sum"
+        hash sha256sum 2>/dev/null || {
+        echo >&2 "SHA checksum utility not available. Either install one (shasum or sha256sum) or run with PRYSM_ALLOW_UNVERIFIED_BINARIES=1."
+        exit 1
+        }
```
Comment (prestonvanloon) [inline prysm.sh:136]: Indentation is still different here. Please fix

Comment (jsvisa) [reply prysm.sh:136]: sorry for my carelessness

**Review by prestonvanloon — APPROVED** [review]

**Comment by prestonvanloon** [comment]
Thank you!

**Pull request merged**